### ISSUE TITLE
Text color on some pages has low contrast on Dark Angel theme

### ISSUE THREAD

**Issue by Ninten-Droid** [VocaDB/vocadb#602]
VocaDB

Not a bug, but more of an accessibility issue. The contrast is too low for text in some areas of the site when using the Dark Angel theme, such as highlighted notifications and thread titles. Every time I go to the discussions page, I feel like I need to switch back to the default theme so I can actually read the thread titles. I would like to request the text color for the Dark Angel theme be changed to something a bit darker to increase contrast and improve accessibility, such as #00c7c7. Alternatively, the white/light colored background on these pages could be changed instead, if that would be better. Please consider this change.

![image2](https://user-images.githubusercontent.com/3254864/85351565-18c64400-b4d2-11ea-92ff-899f3604bc9f.png)
^ The discussions page using the Dark Angel theme.
![image5-00c7c7](https://user-images.githubusercontent.com/3254864/85351592-2bd91400-b4d2-11ea-8511-7827cd63c20e.png)
^ .rightFrame td in DarkAngel.css changed to #00c7c7

 - OS: Microsoft Windows 10 
 - Browser: Google Chrome Version 83.0.4103.106

**Comment by riipah** [comment]
You could do a PR yourself and I can review & approve it.

**Comment by ycanardeau** [comment]
You cannot edit `DarkAngel.css` directly because it is auto-generated by [Web Compiler](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.WebCompiler). Please update [DarkAngel.less](https://github.com/VocaDB/vocadb/blob/master/VocaDbWeb/Content/Styles/DarkAngel.less#L44) instead.

**Comment by andreoda** [comment]
Is the "Dark Angel" theme supposed to be a "dark/black" theme like other websites have it?
The white hurts the eyes when browsing in the night. I think we need a proper dark theme for VocaDB.

Examples of "dark" themes that I use:
- GitHub Dark Theme (Add-On for firefox)
- YouTube Deep Dark (by RaitaroH) (included in the "Enhancer for YouTube" firefox Add-On)
- niconico Darkmode (Add-On for firefox)

**Comment by ycanardeau** [comment]
> I think we need a proper dark theme for VocaDB.

@andreoda Do you know how many years it took for GitHub to finally add a dark mode? isaacs/github#66 Easier said than done.

**Comment by riipah** [comment]
Yeah, dark angel theme was added before "dark theme" was a thing :). It's just quickly done by editing the existing stylesheet. Polishing a stylesheet can be tedious. But I think that "dark angel" could be our official "dark theme".
